### DISCUSSION TITLE
Pull Requests reviews are showing irrelevant changes

### DISCUSSION THREAD

**Discussion by DenverCoder1** [community/community#19391] [Mobile] [answered]
When opening a pull request review, from a branch that was based on an outdated main branch, the changes appearing in the mobile app make it seem like the PR is reverting recently pushed commits (it seems to be showing the diff between the branch and the *current* main, rather than only showing the changes that will be applied by the PR).

This creates a very confusing user experience, since this is different from how it works on the GitHub website.

Example of bug:

On desktop, if you visit the Pull Request https://github.com/nextcord/nextcord/pull/687/files, you should see 1 file changed:

![image](https://user-images.githubusercontent.com/20955511/175676832-1f27083f-0c2e-4e90-bd9b-8d3fe4b4d3bd.png)

On mobile, the same PR shows many changes irrelevant to the PR, making it hard to review:

![image](https://user-images.githubusercontent.com/20955511/175673775-3740c8ec-c2b5-49e9-a61d-16ef95ffa564.png)

Currently this bug is making it very difficult to review Pull Requests using the app since it makes it seem like a lot more was changed than actually was and makes the PR look like it will be undoing other commits.

**Comment by melvincwng** [comment]
@DenverCoder1, FYI you can report bugs to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Reply by DenverCoder1** [reply]
Thanks, I have opened a support ticket there.

**Reply by melvincwng** [reply]
@DenverCoder1, glad to have helped!

**Reply by viceice** [reply]
I've no access to the report, but wanted to makre sure the issue happens on **IOS** and **Android** app.

This makes mobile review nearly impossible

**Comment by Maher-Reven** [comment]
@viceice and flutter, am going through the same sometimes.
@DenverCoder1 please can you update us with your latest news whenever it gets fixed?

**Comment by DenverCoder1** [comment, marked as answer]
I just got an update from GitHub Support this morning

> Jun 28, 2022, 8:50 AM UTC
> 
> I'm just writing to you with an update about the problem you reported with PR reviews on mobile showing irrelevant changes. We've pushed out a fix now, so you should no longer hit this issue!
> 
> Thank you for your patience while we worked on this!

**Reply by Maher-Reven** [reply]
🙏  @DenverCoder1

**Reply by viceice** [reply]
is this fix for iOS and Android?

**Reply by DenverCoder1** [reply]
@viceice It was never specified in the ticket, so I assume, probably both. I've only tested on Android.
